client/components/deposits-overview/style.scss

```diff
@@ -41,7 +40,9 @@
 				}
 			}
 		}
-
+		.wcpay-banner-notice.components-notice {
+			margin: 0 24px;
+		}
```

Comment:
I think this will do the trick for [adding margin-bottom to a single deposit](https://github.com/Automattic/woocommerce-payments/pull/5839#issuecomment-1502647809):
```suggestion
			&:last-child {
				margin-bottom: 16px;
			}
		}
```

<img width="713" alt="image" src="https://user-images.githubusercontent.com/3147296/231052037-44b5872a-bfd3-4bda-af15-c313a8161b7c.png">

Reply:
Oh good catch. Thanks!

Reply:
fixed in [fe6236e](https://github.com/Automattic/woocommerce-payments/pull/5839/commits/fe6236e3e772d350cafbc70728bde5d37ab7882a)